docs/flow/unidist/config.rst

```diff
@@ -0,0 +1,68 @@
+..
+      Copyright (C) 2021-2022 Modin authors
+
+      SPDX-License-Identifier: Apache-2.0
+
+:orphan:
+
+Unidist Configuration Settings
+""""""""""""""""""""""""""""""
+
+Using these configuration settings, the user can tune unidist's behavior. Below you can find
+configuration settings currently provided by unidist.
```

Comment:
I would add that all config setting-up should be done before `unidist.init()`.

Reply:
Added a note below.